### PR TITLE
Merge release/4.1 to develop

### PR DESCRIPTION
**Pull request by elfenheart**
3 conflict, `adapters.less`, `wrangler-modal.less`, `hdinsight/pkg/install.sh`.

fixed UI conflicts. 

On hdinsight install script, took head, and change the `CDAP_TAG` to be `v4.2.0`

### CODE REVIEW

File: cdap-distributions/src/hdinsight/pkg/install.sh [hunk 1]
```diff
@@ -22,9 +22,15 @@
 # The git branch to clone
 CDAP_BRANCH='release/4.1'
 # Optional tag to checkout - All released versions of this script should set this
+<<<<<<< HEAD
 CDAP_TAG='v4.1.0'
 # The CDAP package version passed to Chef
 CDAP_VERSION='4.2.0-1'
+=======
```
Comment (anwar6953) [inline cdap-distributions/src/hdinsight/pkg/install.sh]: I'd vote for the bottom one

Comment (dereklwood) [reply cdap-distributions/src/hdinsight/pkg/install.sh]: basically we should just set it to what will be appropriate after release... ie HEAD version but with tag updated to v4.2.0

**Comment by ajainarayanan** [comment]
UI Changes LGTM 👍

File: cdap-ui/app/tracker/services/my-jump-factory.js [hunk 2]
```diff
@@ -26,7 +26,7 @@ function myJumpFactory($state, myTrackerApi, $rootScope) {
   let corePluginsArtifacts = {
     name: 'core-plugins',
     scope: 'SYSTEM',
-    version: '1.6.0'
+    version: '1.6.1'
```
Comment (anwar6953) [inline cdap-ui/app/tracker/services/my-jump-factory.js]: on develop, I think this is 1.7.0-SNAPSHOT.
https://github.com/caskdata/hydrator-plugins/blob/develop/core-plugins/pom.xml#L26

File: cdap-ui/server/config/cdap-ui-config.json [hunk 3]
```diff
@@ -2,7 +2,7 @@
   "tracker": {
     "artifact": {
       "name": "tracker",
-      "version": "0.4.1",
+      "version": "0.4.0",
```
Comment (anwar6953) [inline cdap-ui/server/config/cdap-ui-config.json]: Can you also update this to 0.5.0-SNAPSHOT, for develop.

**Comment by anwar6953** [comment]
LGTM

**Pull request merged**